_posts/2023-01-11-install-and-manage-applications.md

```diff
@@ -11,555 +11,245 @@ published: true
+### Removing Applications
 
-- Alternatively, you can install the applications by entering the Alpine Linux container using the following commands:-
+You can remove the installed Flatpaks and Snaps from the `Installed` panel.
 
-```bash
-apx enter --apk
-sudo apk add <packages>
-exit
-apx export --apk <packages>
-```
+(**_Note_**: Some native applications installed on the host cannot be removed using this method)
 
-- You can export the installed binary to use it with the host directly using the following command:-
+## Manage AppImages
 
-```bash
-apx export --apk --bin <package>
-```
+### Installing AppImages
 
-### Uninstalling Applications from the Alpine Linux Container
+You can download AppImages from project pages on GitHub. Once downloaded, they won't have to be installed since they are executable binaries.
 
-- You can uninstall applications from the Alpine Linux container using the following command:-
+### Running AppImages
 
-```bash
-apx remove --apk <package>
-```
+If you have enabled AppImage in the first setup, you can run AppImages graphically by following these steps:-
 
-- This command will automatically detect and remove the desktop file entry.
+- Right-click on the file, then click on **Properties**.
+    
+![AppImages Properties](/assets/uploads/Applications_Post/appimages-nautilus-properties.webp)
+    
+- Enable the **Executable as Program** option.
 
-- Alternatively, you can uninstall the applications by entering the Alpine Linux container using the following commands:-
+You can now run the AppImage by right-clicking it and pressing run or by pressing enter/return key on your keyboard.
 
-```bash
-apx enter --apk
-sudo apk del <packages>
-exit
-apx unexport --apk <packages>
-```
-
-- You can unexport the exported binary from the host using the following command:-
+Alternatively, you can open AppImages from the terminal with the following commands:-
 
 ```bash
-apx unexport --aur --bin <package>
+cd <directory>
+chmod +x <file>.appimage
+./<file>.appimage
 ```
 
-### Installing Applications in the openSUSE Container
+**_Note_**:- `chmod +x <file>.AppImage` makes the file executable.
 
-- You can install applications in the openSUSE container using the following command:-
+## Managing Flatpaks through CLI
 
-```bash
-apx install --zypper <packages>
-```
+If you enabled Flatpak in the First Setup, you won't have to do anything. If you have disabled Flatpak in the First Setup, follow the guide [**here**](/2022/12/09/install-flatpaks.html).
 
-- This command will automatically detect the desktop file entry in the package and add it to the Application menu and Vanilla Control Center.
+### Installing Flatpaks
 
-- Alternatively, you can install the applications by entering the openSUSE container using the following commands:-
+You can install Flatpaks with the following command:-
 
 ```bash
-apx enter --zypper
-sudo zypper install <packages>
-exit
-apx export --zypper <packages>
+flatpak install <remote> <application-id>
```

Comment:
```suggestion
flatpak install <application-id>
```
I would leave out the `<remote>` part since not everyone knows what to put there and users will be prompted for the remote to use if there are matches in multiple remotes.